package.json

```
@@ -55,6 +55,9 @@
     "protractor": "~7.0.0",
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
-    "typescript": "~3.8.3"
+    "typescript": "3.8.3"
+  },
+  "overrides": {
+    "ip": { ".": "npm:neoip@2.1.0" }
```

I believe this can also be simplified as:

```suggestion
    "ip": "npm:neoip@2.1.0"
```